<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>698</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Penjualan</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Tanggal</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="dtpTanggal">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>30</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>70</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>keterangan</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="txtKeterangan">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>60</y>
     <width>481</width>
     <height>71</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="tblObat">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>271</y>
     <width>671</width>
     <height>321</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Kode Obat</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nama Obat</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Jumlah Obat</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Harga Obat</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Sub Total</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="btnTambahObat">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>200</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Cari Obat</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSimpan">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>640</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Simpan</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnKeluar">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>640</y>
     <width>101</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Keluar</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtKodeObat">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>230</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtNamaObat">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>230</y>
     <width>191</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtJml">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>81</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtHargaObat">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>230</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInsertObat">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>220</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Tambah Obat</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>600</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Total Biaya</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTotalBiaya">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>600</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Total</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTotal">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
